hibernate-testing/src/main/java/org/hibernate/testing/orm/junit/SettingProvider.java

```diff
@@ -15,5 +15,5 @@
 	}
 
 	String settingName();
-	Class<? extends Provider<?>> provider();
+	Class<? extends Provider<?>> providerClass();
```

Comment:
Please revert to the shorter name. It doesn't add any value and will reduce the amount of files in the PR.